### PR TITLE
webdav lock protection without skeleton

### PR DESCRIPTION
**Pull request by haribhandari07**
## Description

Do not use skeleton files for ```webUIWebdavLockProtection``` to speed up CI
## Related Issue

Part of https://github.com/owncloud/QA/issues/622

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: tests/acceptance/features/webUIWebdavLockProtection/move.feature [hunk 1]
```diff
@@ -44,9 +48,11 @@ Feature: Locks
       | shared    |
 
   Scenario Outline: moving a file into a locked folder
+    Given user "brand-new-user" has created folder "/simple-empty-folder"
+    And user "brand-new-user" has uploaded file "filesForUpload/lorem.txt" to "/lorem.txt"
     Given user "brand-new-user" has locked file "/simple-empty-folder" setting following properties
```
Comment (paurakhsharma) [inline tests/acceptance/features/webUIWebdavLockProtection/move.feature]: s/Given/And

**Review by dpakach — COMMENTED** [review]
LGTM :+1:

**Review by paurakhsharma — COMMENTED** [review]
Looks good to me :+1:

**Review by phil-davis — APPROVED** [review]

**Comment by HariBhandari07** [comment]
Backport on #35574

**Pull request merged**